packages/vdom/vdom.0.3/opam

```diff
@@ -0,0 +1,49 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "DOM and VDOM for OCaml"
+description: """
+This package contains:
+- OCaml bindings to DOM and other client-side Javascript APIs (using gen_js_api).
+- An implementation of the Elm architecture, where the UI is specified as a functional "view" on the current state."""
+maintainer: ["Alain Frisch <alain.frisch@lexifi.com>"]
+authors: [
+  "Alain Frisch <alain.frisch@lexifi.com>"
+  "Marc Lasson <marc.lasson@lexifi.com>"
+  "Aurélien Saue <aurelien.saue@lexifi.com>"
+]
+homepage: "https://github.com/LexiFi/ocaml-vdom"
+license: "MIT"
+bug-reports: "https://github.com/LexiFi/ocaml-vdom/issues"
+depends: [
+  "ocaml" {>= "4.10"}
+  "dune" {>= "3.0"}
+  "gen_js_api" {>= "1.0.7"}
+  "ojs"
+  "js_of_ocaml-compiler"
+  "odoc"
```

Comment:
@AurelienSaue Does vdom really depend odoc?

Reply:
It does not seem the case, however it was done intentionally: https://github.com/LexiFi/ocaml-vdom/pull/43

Reply:
It doesn't, the `{with-doc}` got lost. My apologies!

Reply:
@mseri should I PR the fix?

Reply:
@AurelienSaue no thanks, I have just done it here: https://github.com/ocaml/opam-repository/pull/24713
Please make a PR to fix the issue upstream if it is still there

Reply:
I did, thank you!